### PR TITLE
EOS-23134 cortx_setup config get doesn't return any error message when executed on non-set key

### PR DESCRIPTION
**Pull request by zahidsk**

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
Why are we changing durability related key info?

File: lr-cli/cortx_setup/commands/storageset/config/durability.py [hunk 1]
```diff
@@ -77,7 +75,7 @@ def run(self, **kwargs):
             )
 
             ss_name = Conf.get(index, f'cluster>{cluster_id}>storage_set[0]>name')
-
+            d_data = Conf.get(index, f'cluster>{cluster_id}>storage_set[0]>durability')
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/storageset/config/durability.py:78]: Change `d_data` to `durability_info`

File: lr-cli/cortx_setup/commands/storageset/config/durability.py [hunk 2]
```diff
@@ -77,7 +75,7 @@ def run(self, **kwargs):
...
             durability_dict = {durability_type: {}}
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/storageset/config/durability.py:79]: Remove unused durability_dict

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/storageset/config/durability.py (hunk 1 above)
Comment (zahidsk) [reply lr-cli/cortx_setup/commands/storageset/config/durability.py]: addressed in respective PR

File: lr-cli/cortx_setup/commands/storageset/config/durability.py (hunk 2 above)
Comment (zahidsk) [reply lr-cli/cortx_setup/commands/storageset/config/durability.py]: removed

File: lr-cli/cortx_setup/commands/config/get.py [hunk 3]
```diff
@@ -50,6 +50,11 @@ def run(self, key=None):
         try:
             val_fr_conf = Conf.get(index, key)
             self.logger.debug(f"Config value for '{key}' : {val_fr_conf}")
+            if not val_fr_conf:
+                self.logger.warning(
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/config/get.py:54]: This is log warning how is it printed on screen?

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/config/get.py [hunk 4]
```diff
@@ -50,6 +50,11 @@ def run(self, key=None):
         try:
             val_fr_conf = Conf.get(index, key)
             self.logger.debug(f"Config value for '{key}' : {val_fr_conf}")
+            if not val_fr_conf:
+                self.logger.warning(
+                   f"Given key '{key}' is possibly not found "
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/config/get.py:55]: Change this message:
`Configuration key {key} does not exist`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/config/get.py (hunk 4 above)
Comment (zahidsk) [reply lr-cli/cortx_setup/commands/config/get.py]: Updated

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Pull request merged**